src/scripts/main.js

```diff
@@ -1,3 +1,17 @@
 'use strict';
 
 // write your code here
+
+
+const [...ppl] = document.querySelectorAll('.population');
```

Comment:
```suggestion
const [...population] = document.querySelectorAll('.population');
```
don't use short forms. use meaningful names for variables instead